examples/single_tier_cache/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
nit. `Copyright (c) Meta Platforms, Inc. and affiliates.`

Reply:
Done